_pages/who_uses_login.md

```diff
@@ -0,0 +1,37 @@
+---
+title: meta.who-uses-login.title
+description: meta.who-uses-login.description
+permalink: /who-uses-login/
+twitter_card: large
+image: /assets/img/login-gov-600x314.png
+---
+
+{% capture heading %}
+  {% t who-uses-login-page.hero.heading %}
+{% endcapture %}
+
+{% capture text %}
+  {% t who-uses-login-page.hero.text %}
+{% endcapture %}
+
+{% include hero.html class="who-uses-login" heading=heading text=text col_class="sm-col-6" %}
+
+<div class="bg-lightest-blue">
+  <div class="container who-uses-login">
+    <div class="partners bg-lightest-blue">
+      {{ site.translations[site.lang]["who-uses-login-page"]["partners"] | replace: 'site.baseurl', site.baseurl | markdownify }}
+    </div>
+  </div>
+  <div class="bg-white">
+    <div class="container who-uses-login">
+      <div class="security">
+        {{ site.translations[site.lang]["who-uses-login-page"]["security"] | replace: 'site.baseurl', site.baseurl | markdownify }}
+      </div>
+    </div>
+  </div>
+</div>
+
+{% capture banner_content %}
+  <p><a class="learn-account-creation link" href="{{ site.baseurl }}/create-an-account">{% t banner.one-account-for-govt.learn %}</a></p>
+{% endcapture %}
+{% include one_account_banner.html content=banner_content %}
```

Comment:
It appears from the designs that the banner is intended to be hidden on mobile devices?

```suggestion
<div class="display-none tablet:display-block">
  {% include one_account_banner.html content=banner_content %}
</div>
```

Reply:
Addendum: From what I can tell, BassCSS is still used on the demo site, meaning the above suggestion won't work, due to the same conflict identified in BassCSS transition technical discovery.

https://github.com/basscss/basscss/blob/v7.0.4/css/basscss.css#L463

Next best option might be to use [`at-media-max` USWDS helper](https://designsystem.digital.gov/design-tokens/spacing-units/#using-spacing-unit-tokens), as mentioned at https://github.com/18F/identity-idp/pull/4194/files#r488066341.

Reply:
Talked with @aduth and we agreed that adding the `at-media-max` mixin is the best option. He will be adding a commit suggestion for this PR.